ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/saas/searchresults/resultitem-template.html

```diff
@@ -0,0 +1,11 @@
+<sly data-sly-template.resultItem="${@ model, count}">
+  <div class="result-item" result-item>
+    <a href="${model.url}">
+      <div class="result-meta">
+        <h3>${'{0}. {1}' @ format=[count,model.title], context='unsafe'}</h3>
+        <div class="result-info"><cite class="result-path">${model.url}</cite></div>
+        <div class="result-outline"><span class="" meta-description>${model.description @ context='unsafe'}</span></div>
```

Comment:
context unsafe? please change to at least `html`

Reply:
The context 'unsafe' is intentionally specified. By default sightly resolves context to 'html', but this causes the highlighting tag to be escaped.